enhancements/update/update-blocker-lifecycle/README.md

```diff
@@ -5,17 +5,19 @@ authors:
 reviewers:
   - "@LalatenduMohanty"
 approvers:
-  - TBD
+  - "@sdodson"
+api-approvers:
+  - None
 creation-date: 2020-09-11
-last-updated: 2021-06-17
+last-updated: 2022-04-19
 status: implementable
 ---
 
 # Update-blocker Lifecycle
 
 We occasionally have bugs which impact update success or the stability of the target release.
-When that happens, we protect users by [removing the impacted updates from our update recommendations][graph-data-block].
-This enhancement describes the process used to identify these bugs and determine which edges should be blocked because of them.
+When that happens, we protect users by [removing or qualifying the impacted updates from our update recommendations][graph-data-block].
```

Comment:
I do not think `we protect users by qualifying the impacted updates from our update recommendations` communicates the same thing you want to communicate

Reply:
Reworded to:

```markdown
When that happens, we protect users by [removing update recommendations or qualifying recommendations them with conditional risks][graph-data-block].
```

with af8037bf -> 92525969.